visidata/loaders/postgres.py

```diff
@@ -110,9 +110,13 @@ def openRow(self, row):
 
 # rowdef: tuple of values as returned by fetchone()
 class PgTable(Sheet):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.schema = options.postgres_schema
+
     @asyncthread
     def reload(self):
-        with self.sql.cur("SELECT * FROM " + self.source) as cur:
```

Comment:
Hi @jdormit, thanks for this PR.  Does this still work if `postgres_schema` is an empty string?  (Is it any different to use a default schema rather than "`public`"?)

Also let's use `self.options.postgres_schema` within reload() itself.  That way, the user can change it as a sheet-specific option and then use Ctrl+R to reload.

Reply:
Ah, good point about `postgres_schema` being blank, I hadn't thought about that.

So I updated the PR to reference `self.options.postgres_schema` within the `reload()` method, but when I do that it appears that `self.options.postgres_schema` is always `public` regardless of the `--postgres_schema` option. Any idea what could be causing that? Maybe the `@asyncthread` decorator?

Reply:
I'll push up my updated code, but it doesn't work right now due to ☝️

Reply:
Hmm, so I am probably revealing my limited VisiData knowledge here. But when I run `vd --postgres_schema=my_schema <postgres url>` and look at the global options via `shift-o`, there is no value set for `postgres_schema`. However, when I am looking at the sheet options (`z shift-o`) for the Postgres tables sheet specifically, `postgres_schema` is set to the expected schema. So that would explain why individual table sheets are ending up with the default value. Do you know why `postgres_schema` is getting set as a sheet-specific option instead of a global option?

Reply:
Yes, options set via the CLI are sheet-specific by default, and will only apply to sheets given as CLI args.  Use the `-g` CLI option to set subsequent CLI options as globals. This behavior is intentional, although as it confuses people (sometimes even me!) we may wind up inverting this behavior at some point.

Reply:
Ah understood. Confirmed that passing `-g --postgres_schema=my_schema` works as expected with this patch!